projects/dsp-ui/src/assets/style/_viewer.scss

```diff
@@ -10,8 +10,8 @@
 
 .read-mode-view {
   font: 400 15px/24px sans-serif;
-  p {
-    margin: 0px;
+  .rm-value {
+    display: block;
```

Comment:
I would also add `margin: 0px` inside .rm-value to align some of the more complex value component values with their label

<img width="1098" alt="Screen Shot 2020-09-10 at 11 01 17" src="https://user-images.githubusercontent.com/60604010/92706784-164aec00-f355-11ea-933b-c9a16764ad61.png">

Reply:
done in cda4f6a30af53b31cad89a9b8ff13f525d2c1da4